matching-web/src/main/java/com/graphhopper/matching/http/MatchServlet.java

```diff
@@ -64,6 +64,7 @@
     public void doPost(HttpServletRequest httpReq, HttpServletResponse httpRes)
             throws ServletException, IOException {
 
+    	logger.info("posted");
```

Comment:
Happy to get rid of it - I shouldn't have left it in there. (I couldn't figure out how to get any debug output from running the GUI, and that was me trying, I think.)